bin/generators/gateway/templates/basic/package.json

```diff
@@ -1,5 +1,8 @@
 {
-  "name": "",
+  "name": "express-gateway-instance",
+  "description": "Express Gateway Instance Bootstraped from Command Line",
+  "repository": {},
+  "license": "MIT",
```

Comment:
Don't need a license here.  How the generated gateway gets licensed is up to the user.  Having this here could actually present an internal governance problem.

Reply:
Npm will throw a warning without. In this case, maybe the best thing would be to ask the license using the CLI?

Reply:
We could use `UNLICENSED` or something.